src/Agent/NewRelic/Agent/Core/Config/Configuration.xsd

```diff
@@ -55,6 +55,13 @@
               </xs:documentation>
             </xs:annotation>
             <xs:sequence>
+              <xs:element name="obscuringKey" type="xs:string" minOccurs="0" maxOccurs="1">
+                <xs:annotation>
+                  <xs:documentation>
+                    The key used to deobfuscate the proxy password (optional).
```

Comment:
We should remove the reference to proxy password.  This is a key for any configuration value.